state/subnets.go

```diff
@@ -43,8 +44,8 @@ type subnetDoc struct {
 }
 
 // Life returns whether the subnet is Alive, Dying or Dead.
-func (s *Subnet) Life() Life {
-	return s.doc.Life
+func (s *Subnet) Life() life.Value {
```

Comment:
I'd be inclined to leave this so it is consistent with the other `state` types, such as `Machine` and 'Unit`.